### ISSUE TITLE
Release v2.1.0 (i.e. tag)

### ISSUE THREAD

**Issue by jflemer-ndp** [luca3m/redis3m#22]
Were you planning to make 2.1.0 a "release" (i.e. tag it)?  The version was bumped here in this commit: b417c0abae3edce01bf3c57f911ef4688bf6566c

**Comment by luca3m** [comment]
Yes I did a version bump but it's not finished the release yet. I wanted to finish all the work to remove boost before releasing a new version. By the way there are not so much improvements from master to last tagged version.
